css/styles.css

```diff
@@ -67,8 +67,9 @@ ul li {
 
 #about-me > ul > li > ul > li > ul > li {
   background-color: white;
-  padding: 8px;
+  padding: 8px 3px;
   border-radius: 7px;
+  font-size: 12px;
```

Comment:
- [ ] Could you please take a look at the mobile version? It seems to have some issues after you applied the styling for the desktop version. Thank you!

   ![image](https://github.com/flov3rh/portfolio/assets/107952724/a273a05c-ffdf-4711-8778-d295e04e6e58)